etc/kayobe/kolla/config/prometheus/rabbitmq.rules

```diff
@@ -6,7 +6,7 @@ groups:
 - name: rabbitmq.rules
   rules:
   - alert: RabbitMQNodeDown
-    expr: sum(rabbitmq_build_info{instance!=""}) < 3
+    expr: sum(rabbitmq_build_info{instance!=""}) < {% endraw %}{{ groups['controllers'] | length }}{% raw %}
```

Comment:
I think you could potentially use the rabbitmq group here since the templating is within a raw tag and thus templated by kolla-ansible